backend/apps/core/middleware/perf_tracking.py

```
@@ -0,0 +1,66 @@
+                view_name=view_name,
+                method=request.method,
+                duration_ms=total_duration,
+                db_query_count=query_count[0],
+                db_duration_ms=db_duration[0],
+                cache_hits=0,
+                cache_misses=0,
+                serialization_ms=0,
+                user_id=user_id,
+            )
+        except Exception:
+            pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)